bottomline_recordandreplay/assets/security/rule-Bottomline__Customer_Maintenance_on_VIP.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Bottomline: Customer Maintenance on VIP",
+  "isEnabled": true,
+  "queries": [
+    {
+      "query": "@bottomline.mainframe.activity.resource.code:BNKCSTF and @bottomline.mainframe.activity.customer.id:300842 ",
+      "groupByFields": [
+        "@bottomline.mainframe.activity.usr.id"
+      ],
+      "hasOptionalGroupByFields": false,
+      "distinctFields": [],
+      "aggregation": "count",
+      "name": ""
+    }
+  ],
+  "options": {
+    "keepAlive": 3600,
+    "maxSignalDuration": 86400,
+    "detectionMethod": "threshold",
+    "evaluationWindow": 900
+  },
+  "cases": [
+    {
+      "name": "Customer Maintenance by User",
+      "status": "high",
+      "notifications": [],
+      "condition": "a > 0"
+    }
+  ],
+  "message": "@{{bottomline.mainframe.activity.usr.id}} performed customer maintenance on VIP account",
```

Comment:
Could you improve/detail a bit more the attack with sections in message:
```### Goal
The goal of the attack.

### Strategy
How to tackle/investigate this issue.

### Triage and response
1. List of action to triage.

Reply:
starting simple